indexers/dbms/PostgreSQLIndexer/start.sh

```diff
@@ -0,0 +1,3 @@
+nohup bash -c "/docker-entrypoint.sh postgres &" &&
+sleep 3 &&
+jina pod --uses config.yml $@
```

Comment:
oh I'm surprised this works actually. These should have a `\` at the end of the first two lines